choreographer/src/main/java/eu/arrowhead/core/choreographer/database/service/ChoreographerSessionDBService.java

```diff
@@ -105,11 +105,11 @@ public ChoreographerSession initiateSession(final long planId, final String noti
       		  worklogAndThrow("Initiating plan has been failed", new InvalidParameterException("Plan with id " + planId + " not exists"));
       	  }
       	  
-      	  final String _notifyUri = UriComponentsBuilder.fromUriString(notifyUri).build().toUriString();
```

Comment:
I used optional as a variable name here as well. Sorry. Please rename it to optPlan or something similar.